frontend/src/components/common/Notifications/index.tsx

```diff
@@ -0,0 +1,210 @@
+import { useTypedSelector } from '../../../redux/reducers/reducers';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  notificationItem: {
+    '&.MuiMenuItem-root': {
+      borderBottom: `1px solid ${theme.palette.notificationBorderColor}`,
+      padding: '1rem',
+    },
+  },
+  notificationMessage: {
+    '&': {
+      wordBreak: 'break-word',
+      whiteSpace: 'normal',
+    },
+  },
+  root: {
+    '& .MuiPaper-root': {
+      width: '30vw',
+      minWidth: '300px',
+      maxHeight: '400px',
+    },
+  },
+  errorItem: {
+    '&': {
+      color: theme.palette.error,
+    },
+  },
+}));
+
+function prepareNotificationsList(
+  notifications: Notification[],
+  clickEventHandler: (notification?: Notification) => void
+) {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  if (!notifications || notifications.length === 0) {
+    return (
+      <MenuItem onClick={() => clickEventHandler()} className={classes.notificationItem}>
+        {t(`frequent|You don't have any notifications right now`)}
+      </MenuItem>
+    );
+  }
+
+  function menuItemClickHandler(notification?: Notification) {
+    if (!notification) {
+      return;
+    }
+    notification.seen = true;
+    clickEventHandler(notification);
+  }
+
+  function prepareNotificationLinkComponent(notification: Notification) {
+    if (!notification.url) {
+      return notification.message;
+    }
+    return (
+      <Link component={RouterLink} to={notification.url}>
+        {notification.message}
+      </Link>
+    );
+  }
+
+  return notifications?.map((notification, index: number) => {
+    return (
+      <MenuItem
+        onClick={() => menuItemClickHandler(notification)}
+        key={`${notification}__${index}`}
+        className={classes.notificationItem}
+      >
+        <Grid container spacing={4}>
+          <Grid item direction="column" md={notification.seen ? 12 : 11}>
+            <Grid item md={12}>
+              <Typography style={{ fontWeight: notification.seen ? 'normal' : 'bold' }}>
+                {prepareNotificationLinkComponent(notification)}
+              </Typography>
+            </Grid>
+            <Grid item>{timeAgo(notification.date)} ago</Grid>
+          </Grid>
+          {!notification.seen && (
+            <Grid item md={1}>
+              <Badge variant="dot" color="error"></Badge>
+            </Grid>
+          )}
+        </Grid>
+      </MenuItem>
+    );
+  });
+}
+
+export default function Notifications() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const notifications = useTypedSelector(state => state.ui.notifications);
+  const dispatch = useDispatch();
+  const [events] = Event.useList();
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    console.log('cluster path');
+    if (events && events.length !== 0) {
+      const importantEvents: Notification[] = events
+        .filter((event: KubeEvent) => event.type === 'Normal')
```

Comment:
I think we had talked about showing only warnings by default, right?
Maybe we can ask some of the Kubernetes volks what would make sense to them.

Reply:
oops pushed this by mistake while testing, warning events rarely happen so i changed it to normal for testing, will revert back to normal because we don't wan't to show every normal event out there because they are of no concern to the user

Reply:
It's still showing the "Normal" events.